### PR TITLE
feat(config) add gravatar config object

### PR DESCRIPTION
**Pull request by mihhu**
Should I whitelist the version I'm deprecating or rather remove it altogether from the config?

### CODE REVIEW

File: react/features/base/config/configWhitelist.js [hunk 1]
```diff
@@ -163,6 +163,7 @@ export default [
     'gatherStats',
     'giphy',
     'googleApiApplicationClientID',
+    'gravatar',
```
[the remark this answers is not in the document]
Comment (mihhu) [reply react/features/base/config/configWhitelist.js:166]: Hmm right.. then a separate field `disableGravatar` would be better? @saghul wdyt?

[the remark this answers is not in the document]
Comment (damencho) [reply react/features/base/config/configWhitelist.js:166]: You can whitelist only `gravatar.disabled` as we do for analytics:
https://github.com/jitsi/jitsi-meet/blob/9aea7ff44500cb2b07e9b3362a092e5a45f39afc/react/features/base/config/configWhitelist.js#L17

**Comment by saghul** [comment]
> Should I whitelist the version I'm deprecating or rather remove it altogether from the config?
> 
> 

Don't whitelist it, but add a deprecation note please.

File: react/features/base/config/configWhitelist.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (saghul) [reply react/features/base/config/configWhitelist.js:166]: This.

File: config.js [hunk 2]
```diff
@@ -587,8 +587,17 @@ var config = {
     // corsAvatarURLs: [ 'https://www.gravatar.com/avatar/' ],
 
     // Base URL for a Gravatar-compatible service. Defaults to libravatar.
+    // DEPRECATED! Use `gravatar` instead.
     // gravatarBaseURL: 'https://seccdn.libravatar.org/avatar/',
 
+    // Setup for Gravatar-compatible services.
+    // gravatar: {
+    //     // Defaults to libravatar.
```
[the remark this answers is not in the document]
Comment (mihhu) [reply config.js:595]: Yes, thanks for pointing it out, I'll update above too.

[the remark this answers is not in the document]
Comment (mihhu) [reply config.js:595]: Should I open a PR to change the default base url from here as well? (I'm not up to date with the libravatar change) https://github.com/jitsi/js-utils/blob/1abd0f1af0ae2637f8f336b484a6cd05593d04af/avatar/index.js#L10

**Review by saghul — APPROVED** [review]

File: config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (saghul) [reply config.js]: Yes please.

**Pull request merged**